tools/gdb/dump_insert.py

```diff
@@ -0,0 +1,77 @@
+import gdb, sys
+ 
+# This script walks an insert list in WiredTiger, it does so by traversing the bottom level of the
+# skip list and printing the next pointer array. It dumps the data to "dump.txt" as the content
+# is often too large for the terminal.
+#
+# It takes a WT_INSERT_HEAD structure and a key format. In order to use it in gdb it must first be
+# sourced. Example usage:
+# source dump_insert.py
```

Comment:
Suggested function and script name: `dump_insert_list`

Reply:
[d83f7e4](https://github.com/wiredtiger/wiredtiger/pull/9406/commits/d83f7e4778537d2e8b29ca16cc502d65a2650293)